developer_guides/debugability/logger/index.rst

```diff
@@ -120,3 +121,62 @@ Examples:
   .. code-block:: bash
 
      sof-logger -l ldc_file -d
+
+
+Trace filtering
+***************
+
+It's possible to change current logs level for any instance of component in
+runtime. To do so, use `-F` option, where `options` have
+following format:
+
+   <log_level>=<component1>[, <component2>]
+
+Where *<log_level>* is one of:
+
+- ``c`` / ``critical``
+- ``e`` / ``error``
+- ``w`` / ``warning``
+- ``i`` / ``info``
+- ``d`` / ``debug``
+- ``v`` / ``verbose``
+
+After **=** character, there is a list of component, separated with comma.
+Each component starts with *name* description followed by optional part with
+*instance* description.
+
+List of possible components names comes from UUID declaration,
+see :ref:`uuid-api` for more detailed information.
+Try use ``-d`` flag in logger to list each component names.
+There is also a special name - ``*`` - used to apply given trace
+level to each component.
+
+Instance description may have one of following form:
+
+- ``*`` - each component instance with given *name*
+- ``X`` - only for *name* equal *pipe*, then apply to pipe component
+- ``X.*`` - update each component on selected pipeline *X*
+- ``X.Y`` - component with id *Y* on pipeline *X*
+
+Trace levels changes works in same order as options given in command line,
+and new set overwrites old value. It allows to easily enable logs only for
+selected components, example:
+
+   sof-logger -l ldc_file -t -Fcritical=* -Fverbose="dai*, volume1.1"
```

Comment:
I think critical should be always on. It means FW cannot continue.

Reply:
What about bang filers "!dai*"

Reply:
> I think critical should be always on.

Sure, it's implemented in such a way, I should rephrase this sentence in dox.

> What about bang filers "!dai*"

I've done this like I wrote here https://github.com/thesofproject/sof/pull/2965#discussion_r431035827, asking FW for list of loaded component is more complicated and error prone solution - possible troubles with memory usage for this list in IPC buffers, and more more needed step in communication, one more IPC message to maintain in kernel and FW (perhaps with variable list). Overwriting last value is much more easy to implement, maintain and I don't think it's less comfortable in usage than "!dai*"

Reply:
We should not need to ask the FW for names.
1) topology component names should be available vis sysfs today.
2) system component names should come from LDC.

Reply:
@ktrzcinx thinking more, it would be really nice for logger to list the components
```
sof-logger --list
sys.mem.heap
sys.dma.dw
...
vol.1.2
vol.1.4
...
```